src/attr/_make.py

```diff
@@ -172,26 +171,13 @@ def attrib(
     .. versionadded:: 18.1.0
        ``factory=f`` is syntactic sugar for ``default=attr.Factory(f)``.
     .. versionadded:: 18.2.0 *kw_only*
+    .. versionchanged:: 19.0.1 *convert* removed
```

Comment:
I'm iffy on how this is set - does the merger update this version number at merge time?